dashboard/src/main/resources/css/dashboard.css

```diff
@@ -43,20 +43,84 @@
  }
  
  h1 {
-   font-size: 36pt;
+   font-size: 3em;
  }
  
  h2 {
-   font-size: 30pt;
+   font-size: 2.5em;
  }
  
  h3 {
-   font-size: 24pt;
+   font-size: 2em;
  }
+ 
  th, td {
    padding: 5pt;
  }
  
  pre {
    line-height: normal;
- }
\ No newline at end of file
+ }
+ 
+
+ /* ----- Statistic ----- */
+.statistics {
+    padding-top: 50px;
+}
+
+.container {
+  min-height: 20ex;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-start;
+}
+
+.statistic_item {
+    flex: 0 0 16em;
+    margin: 1em;
+    padding: 20px 30px;
+}
+
+.statistic_item {
+    border: 1px solid #e5e5e5;
+    position: relative;
+    min-height: 180px;
+    overflow: hidden;
+    margin-bottom: 40px;
```

Comment:
There's a strange margin on the left-side of the box.

`margin-left: unset` and `margin-right: 2em` will align the green box with the headers on their left-edges:

![image](https://user-images.githubusercontent.com/28604/51394202-c8da9500-1b07-11e9-933f-47ccb05de076.png)